hail/python/hail/backend/backend.py

```diff
@@ -150,20 +156,26 @@ def get_reference(self, name):
         return json.loads(Env.hail().variant.ReferenceGenome.getReference(name).toJSONString())
 
     def add_sequence(self, name, fasta_file, index_file):
-        scala_object(Env.hail().variant, 'ReferenceGenome').addSequence(name, fasta_file, index_file)
+        scala_object(Env.hail().variant, 'ReferenceGenome').addSequence(
+            name, fasta_file, index_file)
 
     def remove_sequence(self, name):
         scala_object(Env.hail().variant, 'ReferenceGenome').removeSequence(name)
 
     def add_liftover(self, name, chain_file, dest_reference_genome):
-        scala_object(Env.hail().variant, 'ReferenceGenome').referenceAddLiftover(name, chain_file, dest_reference_genome)
+        scala_object(Env.hail().variant, 'ReferenceGenome').referenceAddLiftover(
+            name, chain_file, dest_reference_genome)
 
     def remove_liftover(self, name, dest_reference_genome):
-        scala_object(Env.hail().variant, 'ReferenceGenome').referenceRemoveLiftover(name, dest_reference_genome)
+        scala_object(Env.hail().variant, 'ReferenceGenome').referenceRemoveLiftover(
+            name, dest_reference_genome)
 
     def parse_vcf_metadata(self, path):
         return json.loads(Env.hc()._jhc.pyParseVCFMetadataJSON(path))
 
+    def fs(self):
+        return HadoopFS.get_instance()
```

Comment:
I'd create the HadoopFS in __init__ and just return self.fs here.  We're already tracking a single instance of Backend, so piggy back on that.